suite-native/formatters/src/components/FiatAmountFormatter.tsx

```diff
@@ -21,9 +21,12 @@ export const FiatAmountFormatter = ({
 }: FiatAmountFormatterProps) => {
     const { FiatAmountFormatter: formatter } = useFormatters();
 
-    const isTestnetValue = !!network && isTestnet(network);
```

Comment:
Can you please keep it as variable?

Reply:
What added value does it have?

Reply:
It's easier to understand what `if` condition is for. Here is relatively simple so it's probably fine dropping it but in many cases it helps a lot.